docs/Deploying/Configuration.md

```diff
@@ -2522,3 +2522,27 @@ If not set then the Prometheus `/metrics` endpoint is disabled.
     basic_auth:
       password: "bar"
   ```
+
+#### Packaging
+
+Configurations used to control how Fleet interacts with the packaging server.
```

Comment:
with the (coming soon) packaging server?

(For this iteration of Fleet Sandbox there won't be a packaging server.)